src/SonarScanner.MSBuild.Tasks/Resources.resx

```diff
@@ -220,4 +214,7 @@ Error: {1}</value>
   <data name="WPIF_GeneratingRandomGuid" xml:space="preserve">
     <value>No ProjectGuid has been found in neither the csproj nor the solution (Project {0}). A random one has been generated ({1}).</value>
   </data>
+  <data name="IsTest_NameNotChecked" xml:space="preserve">
+    <value>Regular expression was not set in analysis config file. Project file name will not be checked.</value>
```

Comment:
This would be more useful if it mentioned the name of the property
e.g. _The deprecated regular expression property `sonar.msbuild.testProjectPattern` was not set in the analysis config file. The project file name will not be checked._

Reply:
also, do we log when the `testProjectPattern` is being used? Even with a `low` importance, it would help.

Reply:
Usage of `testProjectPattern` wasn't and isn't logged. I think it's not worth it as we don't need to help users to onboard deprecated feature. And projects classified as test by name will have a log line why it happened. Then it's easy to find out that it's due to this property.